airflow/providers/databricks/operators/databricks.py

```diff
@@ -274,6 +274,7 @@ def __init__(
         idempotency_token: Optional[str] = None,
         access_control_list: Optional[List[Dict[str, str]]] = None,
         wait_for_termination: bool = True,
+        git_source: Optional[Dict[str, str]] = None,
```

Comment:
Can you add a docstring about it on L244

Reply:
Added, 0a264ada2. Could you please have another look to see if that's descriptive enough?